plumbum/commands/processes.py

```diff
@@ -304,6 +303,7 @@ def run_proc(proc, retcode, timeout=None):
 BY_POSITION = object()
 BY_TYPE = object()
 DEFAULT_ITER_LINES_MODE = BY_POSITION
+DEFAULT_BUFFER_SIZE = _INFINITE = float("inf")
```

Comment:
```suggestion
DEFAULT_BUFFER_SIZE = float("inf")
```

Don't think there's a point to keeping this around.